app/views/cards/_form.html.erb

```diff
@@ -8,11 +8,12 @@
     f.input :description,
       input_html: {
         class: 'textile h-100',
-        data: textile_default_data.merge(
+        data: {
           allow_dropdown: @card.new_record?,
           behavior: 'rich-toolbar',
+          paths: editor_paths,
           'rich-toolbar': 'block-code bold field italic link list-ol list-ul table'
```

Comment:
Even though this is not supposed to be in this PR, but do you mind updating the 'rich-toolbar': to rich_toolbar: instead?
`rich_toolbar: 'block-code bold field italic link list-ol list-ul table'`

Reply:
I have nothing against the change but this will affect a bunch of other parts of the app which will bloat this PR. I'm leaning on making this change in a separate PR to keep things simple. Also, this is related to Daniel's comment in the other PR https://github.com/dradis/dradis-ce/pull/626/files/f94f810aed3108ed69baedef20e84440890058e8#r407519151.

Reply:
> Even though this is not supposed to be in this PR

That's part of the "don't do too many things in a single PR issue"

Let's not change areas the PR doesn't intend to handle.